ibis/omniscidb/tests/test_client.py

```diff
@@ -106,6 +106,77 @@ def test_union_op(alltypes):
         expr.compile()
 
 
+@pytest.mark.xfail
```

Comment:
why are these xfailed?

Reply:
I've forgotten to note that there are problems in OmniSci side when both view and table are created via `join`:
1) [issue#526](https://github.com/omnisci/omniscidb/issues/526)
2) [issue#542](https://github.com/omnisci/omniscidb/issues/542)

Reply:
That's why I've marked these tests as xfailed.

Reply:
@jreback , can we merge this PR? I think, after OmniSci fixes problems mentioned above by me the xfailed mark can be deleted.

Reply:
@jreback , can you please answer to these questions?
1) can we merge this PR? or
2) Should I delete xfailed mark and remain tests as failed?

I ask because merging process into master is going on, but this PR is hanging.

Reply:
you haven't explained why things are xfailed

Reply:
I had marked these tests as xfailed because I had used the following actions:
joined_tbl = tbl1.inner_join(tbl2, [tbl1.c == tbl2.g, tbl1.d == tbl2.h])
con.create_table(joined_tbl_name, joined_tbl) <- on this line it was occurring the exception raising: Sqlite3 Error: UNIQUE constraint failed: mapd_columns.tableid, mapd_columns.name
I think, this problem is on OmniSci side.
Now, I have deleted xfailed mark and am using con.create_view(joined_tbl_name, joined_tbl) instead of con.create_table(joined_tbl_name, joined_tbl). Exception isn't occurring.